contrib/redwood/plugins.yml

```diff
@@ -76,6 +128,19 @@
   description: |
     Sentry integration for Open edX.
 
+- name: webhooks
+  src: tutor-contrib-webhooks>=18.0.0,<19.0.0
+  url: https://github.com/aulasneo/tutor-contrib-webhooks
+  author: Andrés González <andres@aulasneo.com>
+  maintainer: Andrés González <andres@aulasneo.com>
+  description: |
+    Connect your Open edX instance to external services with this handful plugin.
+    by adding webhooks and webfilters.
```

Comment:
It looks like this sentence was split half-way. Do you want to keep it that way?